mock/py/mockbuild/exception.py

```diff
@@ -55,7 +54,7 @@ def get_class_by_code(exit_code):
     elif exit_code == 1:
         return Error("Unknow error happened.")
     elif exit_code == 2:
-        return Error("Run without setuid wrapper.", 2)
+        return Error("Run without setuid wrapper.")
```

Comment:
Why? Isn't better way to add the exit code in other parts of this if/else?

Reply:
Tihs is a typo.